tsdb_k8s_queries/pod-template.json

```diff
@@ -38,13 +32,15 @@
             "name": "metrics"
           },
           {%- endif -%}{# non-serverless-index-settings-marker-end #}
+            {# Note: index settings which can be applied freely in serverless begin here #}
           "mapping": {
             "total_fields": {
               "limit": "10000"
             }
           },
           "query": {
             "default_field": [
+              "message",
```

Comment:
Same here delete it

Reply:
Resolved in https://github.com/elastic/rally-tracks/pull/553/commits/b1bd321591cd7c59dd64798e9c06056b1c2caa43.